### ISSUE TITLE
Dragging cursor trigger PressInteraction.Release

### ISSUE THREAD

**Issue by frankois944** [MohamedRejeb/compose-rich-editor#356]
Hi,

I'm using the v1.0.0-rc07, compose multiplatform (1.7.0-beta02) on Android, I'm observing the `interactionSource` of the editor.

When I'm moving/dragging the cursor, the event `PressInteraction.Release` is triggered.

I have something like this :

```
   val richTextState = rememberRichTextState()
   val interactionSource = remember { MutableInteractionSource() }

    LaunchedEffect(interactionSource) {
        interactionSource.interactions.collect { interaction ->
            println(interaction)
        }
    }
   RichTextEditor(
         interactionSource = interactionSource,
         state = richTextState
   )
```

**Comment by MohamedRejeb** [comment]
Hi,
Thanks for reporting this issue.
I will check it. Currently, the library is using Compose 1.6.11 so it maybe just an issue with versions.

**Comment by frankois944** [comment]
Also, I'm trying the dev branch and future version of compose 1.7.0, the Editor is much instable on iOS than the 1.6.*.

**Comment by frankois944** [comment]
> Hi, Thanks for reporting this issue. I will check it. Currently, the library is using Compose 1.6.11 so it maybe just an issue with versions.

Yes, that's the reason. I'm using your editor with iOS and using the latest version of compose fix/add some issues, beta issues :).

**Comment by MohamedRejeb** [comment]
Hopefully, I will start publishing releases for Compose alpha/beta/rc versions
